src/main/java/org/jenkinsci/plugins/github/webhook/subscriber/PingGHEventSubscriber.java

```diff
@@ -59,18 +61,23 @@ protected boolean isApplicable(Item project) {
      */
     @Override
     protected void onEvent(GHEvent event, String payload) {
```

Comment:
https://github.com/jenkinsci/github-plugin/blob/93d40692ff3866705175624e93ec584d4ac88132/src/main/java/org/jenkinsci/plugins/github/extension/GHEventsSubscriber.java#L208-L214
so try-catch useless wrapper, undo change and check that parser reports meaningful error.